### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by DoubleLoong-LearningAccount**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by DoubleLoong-LearningAccount — COMMENTED** [review]
It’s just my learning process on GitHub!

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
```
Comment (doubledragon2) [inline index.html:47]:
```suggestion, my suggestions
  </tr>
```

File: index.html [hunk 2]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
...
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
 
 <script>
   // Initialization
```
Comment (doubledragon2) [inline index.html:53]:
```suggestion
  // Initialization
  my suggestions
```

Comment (doubledragon2) [inline index.html:53]:
```suggestion
my suggestions
```

File: index.html [hunk 3]
```diff
@@ -47,10 +47,10 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
 
 <script>
-  // Initialization
+my suggestions
```
Comment (doubledragon2) [inline index.html:53]:
```suggestion
my suggestions
hahaha
```

File: index.html [hunk 4]
```diff
@@ -47,10 +47,10 @@ <h1>Tic Tac Toe</h1>
...
   const boxes = Array.from(document.querySelectorAll("td"));
   const inputs = Array.from(document.querySelectorAll("[type=checkbox]"));
```
Comment (doubledragon2) [inline index.html:55]:
```suggestion
my suggestions
```

**Pull request merged**